search/pool.go

```diff
@@ -42,11 +53,10 @@ func NewDocumentMatchPool(size, sortsize int) *DocumentMatchPool {
 	startBlock := make([]DocumentMatch, size)
 	startSorts := make([]string, size*sortsize)
 	// make these initial instances available
-	i, j := 0, 0
-	for i < size {
+	j := 0
+	for i := 0; i < size; i++ {
```

Comment:
what is the reason we're changing this in this PR?

Reply:
This shouldn't have been part of this change. Let me undo this.